src/main/notifications/index.ts

```diff
@@ -32,19 +32,19 @@ class NotificationManager {
 
         if (!Notification.isSupported()) {
             log.error('notification not supported');
-            return;
+            return {result: 'error', reason: 'notification_api', data: 'notification not supported'};
```

Comment:
Should we consider "codifying" the reasons so that other sides (e.g. web client, server) can understand them? Maybe a common enum type or something could be an option.

Reply:
I'm thinking of doing that once I have a final final decision on how to track everything. I'll probably do that in the phase where I actually start adding metrics around this stuff :P